config/chef_apply.rb

```diff
@@ -0,0 +1,99 @@
+execute "yum-update" do
+  command "yum update -y"
+end
+
+execute "yum-upgrade" do
+  command "yum upgrade -y"
+end
+
+execute "yum-groupinstall" do
+  command "yum groupinstall -y 'Development Tools'"
+end
+
+package "epel-release" do
+  action :install
+end
+
+bash "install_rabbitmq" do
+  user "root"
+  cwd "/tmp"
+  not_if { ::File.exist?("/etc/init.d/rabbitmq-server") }
+  code <<-EOH
+    wget https://www.rabbitmq.com/releases/erlang/erlang-17.4-1.el6.x86_64.rpm
+    wget https://github.com/rabbitmq/rabbitmq-server/releases/download/rabbitmq_v3_5_3/rabbitmq-server-3.5.3-1.noarch.rpm
+    rpm --import https://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+    yum install -y erlang-17.4-1.el6.x86_64.rpm rabbitmq-server-3.5.3-1.noarch.rpm
+  EOH
+end
+
+bash "install_cassandra" do
+  user 'root'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.